Content.Shared/Damage/Systems/SharedStaminaSystem.cs

```diff
@@ -380,10 +396,11 @@ public override void Update(float frameTime)
 
             comp.NextUpdate += TimeSpan.FromSeconds(1f);
 
-            TakeStaminaDamage(
-                uid,
-                comp.AfterCritical ? -comp.Decay * comp.AfterCritDecayMultiplier : -comp.Decay, // Recover faster after crit
-                comp);
+            if (comp.ActiveDrains.Count == 0) // EE - Harpy Flight
```

Comment:
To clarify: This makes it so that if you pass out from an active drain, you don't recover stamina immediately?